### PR TITLE
Dotnet Release

### PR DESCRIPTION
**Pull request by samimusallam**

### CODE REVIEW

File: .github/workflows/release-dotnet.yml [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+    branches: [ main ]
+
+env:
+  PROJECT_PATH: packages/dotnet/Cisco.Opentelemetry.Specifications
+  CSPROJ_PATH: packages/dotnet/Cisco.Opentelemetry.Specifications/Cisco.Opentelemetry.Specifications.csproj
+  PACKAGE_OUTPUT_DIRECTORY: ${{ github.workspace }}/output
+  NUGET_SOURCE_URL: https://api.nuget.org/v3/index.json
+
+jobs:
+  deploy:
```
Comment (haddasbronfman) [inline .github/workflows/release-dotnet.yml:14]: I just would rename this to release-dotnet to match the other cicd files (or change the other files..)

File: .github/workflows/release-dotnet.yml [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+      - name: Install dotnet-bump
+        run: npm install -g dotnet-bump
+            
+      - name: Bump version
+        working-directory: ${{ packages/dotnet/Cisco.Opentelemetry.Specifications }}
+        run: dotnet-bump patch --dry-run --verbose
+      
+#      - name: Pack & Publish
+#        run: |
+#          dotnet clean
```
Comment (haddasbronfman) [inline .github/workflows/release-dotnet.yml:40]: Do we have an option to run it with dry run?

Comment (samimusallam) [reply .github/workflows/release-dotnet.yml:40]: There's no dry run for the publish command

File: .github/workflows/release-dotnet.yml (hunk 1 above)
Comment (samimusallam) [reply .github/workflows/release-dotnet.yml:14]: true! will fix that to match :)

**Review by haddasbronfman — APPROVED** [review]

**Pull request merged**